### PR TITLE
Adds a FileButton to the InspectableTypes

### PR DESCRIPTION
**Pull request by kircher1**
This playground would demo how to use it: https://www.babylonjs-playground.com/#XK769K#5

### CODE REVIEW

File: packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/customPropertyGridComponent.tsx [hunk 1]
```diff
@@ -131,6 +132,20 @@ export class CustomPropertyGridComponent extends React.Component<ICustomProperty
...
                 return <TextLineComponent key={inspectable.label} label={inspectable.label} value={" "} />;
+            case InspectableType.FileButton:
+                return (
+                    <FileButtonLineComponent
+                        key={inspectable.label}
+                        label={inspectable.label}
+                        onClick={
+                            inspectable.callback ||
+                            function () {
+                                console.warn("no call back function added");
```
Comment (RaananW) [inline packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/customPropertyGridComponent.tsx:143]:
```suggestion
                                Tools.Warn("no callback function added");
```
We do our best not to use console (and especially console.warn) directly. We have Warn in Tools, making it possible to abstract Logging in different scenarios.

**Review by RaananW — CHANGES_REQUESTED** [review]
Try to avoid using `console` directly :-)

Also - what is the main reason to have something like that? What's the use-case?

**Comment by RaananW** [comment]
@sebavan - Tools is not imported, it was just a suggestion :-)
But I think it is more important to understand the need for that before

**Comment by sebavan** [comment]
The need is to have a custom tool where you can load file like a custom texture loader for instance

**Review by sebavan — APPROVED** [review]

**Pull request merged**